examples/convergence_study.jl

```diff
@@ -0,0 +1,82 @@
+# # Performing a convergence study
+#
+# This example shows how to perform a convergence study to find the 
+# appropriate kgrid and Ecut for a desired convergence and error tolerance.
+#
+# Generally, to wisely vary kgrid and Ecut, one can perform a convergence study by
+# starting with a reasonable grid or Ecut, picking a property such as energy, and increasing the grid
+# until the property converges – its value does not change much. 
+
+# This study must be performed separately for each value you wish to study (kgrid, Ecut, and even temperature).
+# In this example we will use a Pt system to perform a convergence study for kgrid and Ecut in DFTK.
```

Comment:
If this is related to an ASE example, you might as well link there.

Reply:
It's an example from Sholl's book https://www.amazon.com/Density-Functional-Theory-Practical-Introduction/dp/0470373172